### PR TITLE
Add sync-to-seafowl CLI command

### PR DESCRIPTION
**Pull request by neumark**

### CODE REVIEW

File: splitgraph/commandline/cloud.py [hunk 1]
```diff
@@ -1310,6 +1310,29 @@ def tunnel_c(remote: str, local_address: str, namespace_and_repository: Optional
         start_repository_tunnel(remote, local_address, namespace_and_repository)
 
 
+@click.command("sync-to-seafowl")
+@click.option("--remote", default="data.splitgraph.com", help="Name of the remote registry to use.")
+@click.argument("vdb_id", type=str)
+@click.argument("seafowl_hostname", type=str)
+@click.argument("seafowl_secret", type=str)
```
Comment (mildbyte) [inline splitgraph/commandline/cloud.py:1317]: Could use a password prompt here if the users don't want the secret to end up in their shell history (https://click.palletsprojects.com/en/7.x/options/#password-prompts)

File: splitgraph/commandline/cloud.py [hunk 2]
```diff
@@ -1310,6 +1310,29 @@ def tunnel_c(remote: str, local_address: str, namespace_and_repository: Optional
...
+    """
+
+    from splitgraph.cloud import GQLAPIClient
+
+    client = GQLAPIClient(remote)
+    task_id = client.start_seafowl_sync(vdb_id, seafowl_hostname, seafowl_secret)
+    final_status = wait_for_job(task_id, lambda: client.get_export_job_status(task_id))
+    if final_status.status == "SUCCESS":
+        assert final_status.output
+        return str("Sync complete")
```
Comment (mildbyte) [inline splitgraph/commandline/cloud.py:1331]: Why the return and not `click.echo`?

File: splitgraph/commandline/cloud.py [hunk 3]
```diff
@@ -1310,6 +1310,29 @@ def tunnel_c(remote: str, local_address: str, namespace_and_repository: Optional
...
+
+    """
+
+    from splitgraph.cloud import GQLAPIClient
+
+    client = GQLAPIClient(remote)
+    task_id = client.start_seafowl_sync(vdb_id, seafowl_hostname, seafowl_secret)
+    final_status = wait_for_job(task_id, lambda: client.get_export_job_status(task_id))
+    if final_status.status == "SUCCESS":
+        assert final_status.output
```
Comment (mildbyte) [inline splitgraph/commandline/cloud.py:1330]: Does the Seafowl sync job actually return anything?

Comment (neumark) [reply splitgraph/commandline/cloud.py:1330]: It displays the URL for the parquet export of the tables. That's not something we really need, but a list of successfully synced tables might be useful. I updated the PR to display this on success.

**Pull request merged**